tensorflow/contrib/ios_examples/simple/tf_ios_makefile_example.xcodeproj/project.pbxproj

```diff
@@ -302,14 +306,18 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "com.google.TF-Test";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SEPARATE_STRIP = NO;
 			};
 			name = Debug;
 		};
 		591157B41CF4011D00C31E3A /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CLANG_DEBUG_INFORMATION_LEVEL = default;
```

Comment:
do we want all of these?

For debug_information_level, is the default for Release essentially 'none'?

exceptions and rtti are not needed by tensorflow and at least rtti adds code size .  (although maybe rtti is needed if we are not using proto lite runtime here).